src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -90,7 +90,7 @@ public static class WebScriptHostBuilderExtension
                         loggingBuilder.Services.AddSingleton<ILoggerProvider, AzureMonitorDiagnosticLoggerProvider>();
                     }
 
-                    if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableDiagnosticEventLogging))
```

Comment:
IMHO, the `Enable` version (instead of the new Disable version) of the feature flag name seems less confusing.

Reply:
The disable is just a safeguard in case a customer is facing an issue with the first release. We will remove disable eventually.